src/dmtcpworker.cpp

```diff
@@ -566,7 +566,10 @@ void dmtcp_Alarm_EventHook(DmtcpEvent_t event, DmtcpEventData_t *data);
 
 void DmtcpWorker::eventHook(DmtcpEvent_t event, DmtcpEventData_t *data)
 {
-  static jalib::JBuffer buf(0); // To force linkage of jbuffer.cpp
+  static jalib::JBuffer *buf = NULL;
+  if (buf == NULL) {
```

Comment:
Suggest adding comment:
// Technically, this is a memory leak, but `buf` is static and so it happens only once.